server/commands/roleplay.py

```diff
@@ -163,7 +163,7 @@ def ooc_cmd_rollp(client, arg):
     """
     roll, num_dice, chosen_max, modifiers, Sum = rtd(arg)
 
-    client.send_ooc(f'[Hidden] You rolled {roll} out of {chosen_max}.\nThe total sum is {Sum}.')
+    client.send_ooc(f'[Hidden] You rolled {roll} out of {chosen_max}.' + f'\nThe total sum is {Sum}.' if num_dice > 1 else '')
```

Comment:
Had to encase this line in () after the + sign, otherwise it interpreted the whole part before "if" as what to display if num_dice is more than >